src/mono/IosAppBuilderTasks/IosAppBuilderTask.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Could we not use ios in the name (same code will be used for tvos, right?)

Reply:
Can you suggest a better name? AppleAppBuilderTask?

Reply:
AppleAppBuilder sounds good to me as the folder/dll name. Should we move this under mono/msbuild/ folder ?